awswrangler/lakeformation/_read.py

```diff
@@ -11,7 +11,7 @@
 from awswrangler._config import apply_configs
 from awswrangler._threading import _get_executor
 from awswrangler.catalog._utils import _catalog_id, _transaction_id
-from awswrangler.distributed import RayLogger, ray_remote
+from awswrangler.distributed.ray import RayLogger, ray_remote
```

Comment:
Should this be imported conditionally as well?

Reply:
No, it has never been imported conditionally (even in the current implementation) because all methods and classes in that `__init__` are used as wrapper in both the distributed and non-distributed cases. Those that exclusively used in the distributed case are imported at a deeper level (e.g. `from awswrangler.distributed.ray._distributed import...`)